### ISSUE TITLE
2022-5-27

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#843]
221更新文档中的所有字段：按Ctrl + A。按F9。(2022-4-19)
222Modify content near logits; done; (2022-4-19)
223公式的括号是黑体，数字是Cambria ; nobody cares; (2022-4-19)
224Modify Ba[10] and Kim[11]; done; (2022-4-19)
225Go over Zhang’s second version; done; (2022-4-19)
226Modify 自注意力；check 知网; 
227Change SKDSAM; done; add 机制 after 自注意力; done; (2022-4-19)
228Deal with content; done; (2022-4-20)
229S1: 摘要：必要性->作用；done; (2022-4-20)
